src/public/interfaces.ts

```diff
@@ -384,6 +384,16 @@ export interface OpenConversationRequest {
   */
   conversationId?: string;
 
+  /**
+   * The Id of the channel. This is optional and should be specified whenever a conversation is started or opened in a personal app scope
+   */
+  threadId?: string;
```

Comment:
Is this truly only for channel tabs? Does not work for tabs configured in group chat?

Reply:
Only for channel tabs